requirements-dev.txt

```diff
@@ -154,15 +155,6 @@ sqlparse==0.4.4
     # via
     #   -r requirements.txt
     #   django
-tomli==2.0.1
```

Comment:
Dropping `tomli` and other changes here seem out of step with what happens if you run the requirements compiling script. Is that expected?

Reply:
It happens if `pip-tools` is run on Python 3.11+, as `tomli` isn't needed then. This change is no longer in the PR.

Reply:
:thumbsup: I suspected it might be that. #65 aims to help with that sort of thing.